modules/stringtie/1.0/config/default.yaml

```diff
@@ -0,0 +1,27 @@
+lcr-modules:
+
+    stringtie_run:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
```

Comment:
Does this module _require_ STAR bam files? If yes, perhaps consider renaming to `star_bam` or include a comment here reminding users that only STAR is acceptable.